data-prepper-plugins/s3-source/src/main/java/org/opensearch/dataprepper/plugins/source/s3/ScanObjectWorker.java

```diff
@@ -268,7 +270,7 @@ private void processFolderPartition(final SourcePartition<S3SourceProgressState>
                 sourceCoordinator.deletePartition(folderPartition.getPartitionKey());
                 return;
             }
-
+            LOG.info("No objects to process, giving up partition");
```

Comment:
This may be a little noisy as info log. Can it be debug?

Reply:
Sure. Will change to debug.

Reply:
Fixed.